packages/redux/src/promotionEvaluations/selectors.ts

```diff
@@ -0,0 +1,118 @@
+/**
+ * @module promotionEvaluations/selectors
+ * @category Promotion Evaluations
+ * @subcategory Selectors
+ */
+
+import { getError, getId, getIsLoading, getResult } from './reducer';
+import type {
+  PromotionEvaluationId,
+  PromotionEvaluationItem,
+} from '@farfetch/blackout-client/src/promotionEvaluations/types';
+import type { State } from './types';
+import type { StoreState } from '../types';
+
+/**
+ * Retrieves a list of all the promotion evaluations items available.
+ *
+ * @function
+ *
+ * @param {object} state - Application state.
+ *
+ * @returns {Array} - List of promotion evaluations items.
+ *
+ * @example
+ * import { getPromotionEvaluationItems } from '@farfetch/blackout-redux/promotionEvaluations';
+ *
+ * const mapStateToProps = state => ({
+ * promotionEvaluationsItems: getPromotionEvaluationItems(state)
+ * });
+ */
+export const getPromotionEvaluationItems = (
+  state: StoreState,
+): State['result'] => getResult(state.promotionEvaluations);
+
+/**
+ * Retrieves a promotion evaluation item given an id.
+ *
+ * @function
+ *
+ * @param {object} state - Application state.
+ * @param {string} id - Promotion evaluation item identifier to retrieve.
+ *
+ * @returns {object | undefined} - Promotion evaluations item.
+ *
+ * @example
+ * import { getPromotionEvaluationItemById } from '@farfetch/blackout-redux/promotionEvaluations';
+ *
+ * const mapStateToProps = (state, { id } ) => ({
+ *   promotionEvaluationsItem: getPromotionEvaluationItemById(state, id)
+ * });
+ */
+export const getPromotionEvaluationItemById = (
+  state: StoreState,
+  id: PromotionEvaluationId,
+): PromotionEvaluationItem | undefined => {
+  const promotionEvaluationsItems = getResult(state.promotionEvaluations);
+
+  return promotionEvaluationsItems?.find(item => item.id === id);
+};
+
+/**
+ * Retrieves the promotion evaluation id stored.
+ *
+ * @function
+ *
+ * @param {object} state - Application state.
+ *
+ * @returns {Array} - List of promotion evaluations items.
```

Comment:
I believe this is not correct, right? 
```suggestion
 * @returns {string} - Promotion Evaluation id.
```